.pre-commit-config.yaml

```
@@ -0,0 +1,59 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+default_install_hook_types: [pre-push]
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-shebang-scripts-are-executable
+      - id: check-merge-conflict
+      # - id: check-symlinks
+      - id: check-toml
+      - id: check-xml
+      - id: check-yaml
+      - id: debug-statements
+      - id: destroyed-symlinks
+      - id: detect-private-key
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: fix-byte-order-marker
+      - id: mixed-line-ending
+      - id: requirements-txt-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/ansible/ansible-lint
+    rev: v25.1.0
+    hooks:
+      - id: ansible-lint
+        additional_dependencies:
+          - ansible
```

ansible-lint is quite slow, making it not very nice for an actual pre-commit hook. How about:

```suggestion
        additional_dependencies:
          - ansible
        stages: [manual]
```

? (The `SKIP` directive can be removed then, I believe.)

---

It is slow, but I think it defeats the purpose of pre-commit if it is not executed automatically (I tend to forget to run all the checks before pushing!). Would that be acceptable?